### ISSUE TITLE
Notebooks do not automatically open in slides format

### ISSUE THREAD

**Issue by markroepke** [uc-python/intro-python-datasci#15]
Notebooks do not currently open in slides format. If students are going to be the primary users, it would make sense for that to be the case.

From the RISE documentation:

*You can setup your notebook to start immediately with the slideshow view using the autolaunch config option.*

```
{
  ...
  "rise": {
    "autolaunch": true,
    "start_slideshow_at": "beginning"
  }
}
```

**Comment by markroepke** [comment]
The above config should be updated slightly. While `"autolaunch": true` does set up the notebook to automatically launch upon opening, the `"start_slideshow_at": "beginning"` sets up the notebook presentation to always launch at the beginning even when launching from notebook mode and another cell is highlighted.

The following should be added to each notebook's metadata:

```
{
  ...
  "rise": {
    "autolaunch": true
  }
}
```

**Comment by markroepke** [comment]
The following notebooks need to be updated:

- [x] Introduction
- [x] Python and Jupyter Overview
- [x] Fundamentals
- [x] Packages, Modules, Methods, Functions
- [x] Importing Data
- [x] Selecting and Filtering Data
- [x] Working with Columns
- [x] Review
- [x] Summarizing Data
- [x] Summarizing Grouped Data
- [x] Joining Data
- [x] Exporting Data
- [x] Visualizing Data

**Comment by markroepke** [comment]
Many of the above were addressed via 6c1af94a5186fc313f302f8316c87b8943d09265
